### PR TITLE
Update readme to human readable

### PR DESCRIPTION
**Pull request by resouer**
Make this repo human readable and easier to follow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,64 +1,73 @@
...
 
-More detailed instructions can be found in the [Crossplane docs]( https://crossplane.io/docs/v0.8/install-crossplane.html).
-
-* Install OAM Core workload and trait controllers
+```console
+git clone git@github.com:crossplane/addon-oam-kubernetes-local.git
+cd ./addon-oam-kubernetes-local
+```
 
+#### Install cert manager
```
Comment (resouer) [inline README.md]: Not sure if we still need this part.

Comment (wonderflow) [reply README.md]: I think we have to remove webhook along with cert manager if we want to change K8s version requirement to 1.16

Comment (resouer) [reply README.md]: webhook is already disabled, this part can be optional.

File: README.md [hunk 2]
```diff
@@ -1,64 +1,78 @@
...
+Here's a minimal version:
+```console
+kubectl create namespace crossplane-system
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install crossplane --namespace crossplane-system crossplane-alpha/crossplane
 ```
-kubectl create namespace cert-manager
 
-kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v0.14.0/cert-manager.yaml
+Please feel free to ignore this warning, which is cause by helm v2 compatibility issue:
```
Comment (ryanzhang-oss) [inline README.md]:
```suggestion
Please feel free to ignore this warning which is caused by helm v2 compatibility issue: 
```

File: README.md [hunk 3]
```diff
@@ -1,64 +1,78 @@
-# oam-kubernetes-implementation
+# OAM Controllers
 
-This OAM Kubernetes implementation allows an application developer/operator to compose a cloud-native application following the
- OAM v1alpha2 spec and run it on a kubernetes cluster.  
+This repo contains Kubernetes controllers for [core](https://github.com/oam-dev/spec/blob/master/3.workload.md#core-workload) and [standard](https://github.com/oam-dev/spec/blob/master/3.workload.md#standard-workload) OAM workloads/traits/scopes. 
 
-## Prerequisites
+> **👷 WARNNING:** This repo is designed to be used by **OAM workloads/traits/scopes developers.**
+> If you are looking for end user experience of OAM application on Crossplane, you probably want to check its
```
Comment (ryanzhang-oss) [inline README.md]:
```suggestion
>  We would suggest you check  [**getting started guide&rarr;**](https://crossplane.io/docs/v0.11/getting-started/install-configure.html) if you are looking for an end-user experience of running OAM applications on the Crossplane.
```

File: README.md [hunk 4]
```diff
@@ -1,64 +1,78 @@
...
-## Getting started
+- Kubernetes v1.16+
+- Helm 3
+- [Crossplane](https://github.com/crossplane/crossplane) v0.11+ installed
 
-The functionality of this addon can be demonstrated with the following steps:
+## (Optional) In case you don't know how to install Crossplane
 
-* Install cert manager
+Here's a minimal version:
```
Comment (ryanzhang-oss) [inline README.md]:
```suggestion
Here's an easy three steps version:
```

File: README.md [hunk 5]
```diff
@@ -1,64 +1,78 @@
...
+
+```console
+git clone git@github.com:crossplane/addon-oam-kubernetes-local.git
+cd ./addon-oam-kubernetes-local
 ```
 
-More detailed instructions can be found in the [Crossplane docs]( https://crossplane.io/docs/v0.8/install-crossplane.html).
+#### (Optional) Enable webhook and install cert manager
+
+In OAM `ManualScalerTrait` controller, it includes a webhook to [validate its workload reference](https://github.com/crossplane/addon-oam-kubernetes-local/blob/757d1922a5266e775b1f131af7da4fb6cbc1a037/pkg/webhooks/manualscalertrait_webhook.go). This webhook [is disabled by default](https://github.com/crossplane/addon-oam-kubernetes-local/blob/42e82c49fb679df4e295802b4727c25faaad4d24/charts/oam-core-resources/values.yaml#L6) in OAM controllers.
```
Comment (ryanzhang-oss) [inline README.md]:
```suggestion
The OAM `ManualScalerTrait` controller includes a sample webhook to [validate the manual scalar trait](https://github.com/crossplane/addon-oam-kubernetes-local/blob/757d1922a5266e775b1f131af7da4fb6cbc1a037/pkg/webhooks/manualscalertrait_webhook.go). This webhook [is disabled by default](https://github.com/crossplane/addon-oam-kubernetes-local/blob/42e82c49fb679df4e295802b4727c25faaad4d24/charts/oam-core-resources/values.yaml#L6) in its helm chart.
```

File: README.md [hunk 6]
```diff
@@ -1,64 +1,78 @@
...
 
-* Install OAM Core workload and trait controllers
+If you want to enable it, the cert manager will be needed to secure the webhook server:
```
Comment (ryanzhang-oss) [inline README.md]:
```suggestion
You need to install a cert-manager to provide self-signed certifications if you want to play with the webhooks.
```

File: README.md [hunk 7]
```diff
@@ -1,64 +1,78 @@
...
 NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
 example-appconfig-workload-deployment   10/10   10           10          8m11s
 ```
 
 And a service looking like below
-```
+```console
 kubectl get services
 AME                                            TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
 example-appconfig-workload-deployment-service   ClusterIP   10.96.78.215   <none>        8080/TCP   8m28s
```
Comment (ryanzhang-oss) [inline README.md]: The default serviceType  is NodeType now

**Comment by resouer** [comment]
All comments addressed!

**Review by ryanzhang-oss — APPROVED** [review]

**Review by wonderflow — APPROVED** [review]

**Comment by ryanzhang-oss** [comment]
please rebase with master

**Comment by wonderflow** [comment]
@resouer Maybe you need rebase first to fix the CI.

**Comment by resouer** [comment]
Just rebased and this should be ready to merge.

**Review by artursouza — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (artursouza) [reply README.md]: I tested on minikube and it works with K9s version 1.16.2+, it does not work with 1.16.0 or 1.16.1 due to errors on Crossplane installation on those versions.

**Pull request merged**